### PR TITLE
authentication done

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
1. Login page
2. Signup Page
3. Login with dummy credentials
4. navbar private routes

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good folder structure
- impressive code

File: src/components/navbar.jsx [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 
 export const Navbar = () => {
+    const token = localStorage.getItem("token");
+    const { cartState } = useCart();
+    const { wishlistState } = useWishlist();
+    const { cartProductsCount } = cartState;
+    const { wishlistProductsCount } = wishlistState;
+
+    const logoutHandler = () => {
+        localStorage.setItem("token","")
```
Comment (SanketDhabarde) [inline src/components/navbar.jsx]: You can use `localStorage.removeItem("token")`

**Review by SanketDhabarde — APPROVED** [review]
- Code looks clean
- UI is good
- good folder structure

File: src/components/navbar.jsx (hunk 1 above)
Comment (govardhanreddyc79) [reply src/components/navbar.jsx]: Modified. Thanks

**Pull request merged**